src/arch/arm/64/machine/fpu.c

```diff
@@ -8,6 +8,9 @@
 #include <arch/machine/fpu.h>
 #include <mode/model/statedata.h>
 
+#define FPU_NOT_IMPLEMENTED (0xF)
+#define IS_FPU_ENABLED(id_aa64pfr0, shift) (((id_aa64pfr0 >> shift) & MASK(4)) != FPU_NOT_IMPLEMENTED)
```

Comment:
macro parameters must be put in brackets to make them atoms.
```suggestion
#define IS_FPU_ENABLED(id_aa64pfr0, shift) ((((id_aa64pfr0) >> (shift)) & MASK(4)) != FPU_NOT_IMPLEMENTED)
```